thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/FetchAnomaliesInRangeAndOutputCSV.java

```diff
@@ -0,0 +1,193 @@
+package com.linkedin.thirdeye.tools;
+
+import com.linkedin.thirdeye.client.DAORegistry;
+import com.linkedin.thirdeye.datalayer.bao.AnomalyFunctionManager;
+import com.linkedin.thirdeye.datalayer.bao.MergedAnomalyResultManager;
+import com.linkedin.thirdeye.datalayer.bao.RawAnomalyResultManager;
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
+import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import com.linkedin.thirdeye.datalayer.dto.RawAnomalyResultDTO;
+import com.linkedin.thirdeye.datalayer.util.DaoProviderUtil;
+import com.linkedin.thirdeye.api.DimensionMap;
+import com.linkedin.thirdeye.integration.AnomalyApplicationEndToEndTest;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.HashSet;
+import org.joda.time.DateTime;
+import org.joda.time.Days;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.ISODateTimeFormat;
+
+public class FetchAnomaliesInRangeAndOutputCSV {
+  private static final String DEFAULT_OUTPUT_FOLDER = "/home/ychung/Desktop/";
+
+  private static AnomalyFunctionManager anomalyFunctionDAO;
+  private static MergedAnomalyResultManager mergedAnomalyResultDAO;
+  private static RawAnomalyResultManager rawAnomalyResultDAO;
+
+  public String dimensionString(DimensionMap dm){
```

Comment:
Exactly same method is defined in another class. If it needs to be reused, create a Utils/Helper class with static reusable methods